features/lockscreen/impl/src/main/kotlin/io/element/android/features/lockscreen/impl/pin/model/PinEntry.kt

```diff
@@ -50,14 +50,36 @@ data class PinEntry(
         return copy(digits = newDigits.toPersistentList())
     }
 
+    fun deleteLast(): PinEntry {
```

Comment:
Not from this PR, but for clarity, maybe rename the fun `fun empty(size: Int)` in the companion to `create`, or `createEmpty`?